src/scripts/main.js

```diff
@@ -1,3 +1,172 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const employeeTable = document.querySelector('table');
+  let currentSort = { column: null, order: 'asc' };
+  let selectedRow = null;
+
+  const headers = document.querySelectorAll('th');
```

Comment:
You are querying all 'th' elements on the page. It's better to limit the scope of selection to your table to avoid any potential issues if other tables are added to the page in the future. Use 'employeeTable.querySelectorAll('th')' instead.